script/fawe.js

```diff
@@ -29,7 +29,7 @@ var projects = {
 		],
 		"config_path": "plugins"
 	},
-	"Spigot 1.16.5 - 1.18.2": {
+	"Spigot 1.16.5": {
```

Comment:
I'm guessing the link for this is probably able to be updated to https://ci.athion.net/job/FastAsyncWorldEdit/588/ as I believe the commit after is what effectively broken 1.16.5 for real

Reply:
done